FakeItEasy.sln

```diff
@@ -1,26 +1,32 @@
-﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 2013
-VisualStudioVersion = 12.0.31101.0

```

Comment:
Yeah, I did hover over this for a few seconds before deciding to let it past. It's because I edited the solution inside VS2015 and then hit "save". Thing is, `MinimumVisualStudioVersion = 10.0.40219.1` (below) hasn't changed. AFAIK the first line `# Visual Studio 14` is just a comment which says which version saved the file, and the second line `VisualStudioVersion = 14.0.24720.0` is just a hint to the Visual Studio Version Selector as to which version to prefer when the sln is shelled out.